CMakeLists.txt

```diff
@@ -1,6 +1,10 @@
 cmake_minimum_required(VERSION 3.16)
```

Comment:
Do we (or any of our dependencies) use any features from 3.16, or could we potentially mark this compatible with a older release?

Reply:
Tensorflow requires this

Reply:
Okay. Just wanted to make sure this is set for a reason and not just arbitrarily.